pkg/explorer/elements/service.go

```diff
@@ -0,0 +1,90 @@
+package elements
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/tdex-network/tdex-daemon/pkg/explorer"
+)
+
+type elements struct {
+	client *RPCClient
+}
+
+// NewService returns and Elements based explorer
```

Comment:
Being a public package (ie. GoDoc self generates documentation out of it) I would document a bit more the public methods (what they expect and what they actually do)

ie. here we are opnening a insecure no-TLS connection by default, that's not obvious for end user

and so on in other methods :pray: